analyses/create-subset-files/01-get_biospecimen_identifiers.R

```diff
@@ -575,10 +580,14 @@ snv_index <- stringr::str_which(names(biospecimen_ids_for_subset), "snv")
 biospecimen_ids_for_subset <- biospecimen_ids_for_subset %>%
   purrr::modify_at(snv_index, ~ append(.x, c(tp53_dnaseq, nf1_dnaseq)))
 
-## add non-GATK samples to cnv file 
-  cnv_index <- stringr::str_which(names(biospecimen_ids_for_subset), "cnv")
+## for Hope tumor only SNV file, add all samples for testing
+Hope_snv <- read_tsv(file.path(data_directory, "snv-mutect2-tumor-only-plus-hotspots.maf.tsv.gz")) %>% 
```

Comment:
```suggestion
tumor_only_snv <- read_tsv(file.path(data_directory, "snv-mutect2-tumor-only-plus-hotspots.maf.tsv.gz")) %>% 
```
prefer this, as we will add more tumor only later